src/games/chess/board.tsx

```diff
@@ -171,7 +171,8 @@ export class Board extends React.Component<IBoardProps, {}> {
 
   _getGameOver() {
     // Online Multiplayer
-    if (this.props.gameArgs && this.props.gameArgs.mode === GameMode.OnlineFriend) {
+    if (this.props.gameArgs &&
+      (this.props.gameArgs.mode === GameMode.OnlineFriend || this.props.gameArgs.mode === GameMode.AI)) {
```

Comment:
add a line before the if:
```
const mode = this.props.gameArgs.mode
````
and then replace with (mode === GameMode.OnlineFriend || mode === GameMode.AI)
this will be much more legible

Reply:
> add a line before the if:
> ```const mode = this.props.gameArgs.mode```

This alone won't work because gameArgs could be undefined.  Yay for ternary operators!  I'm liking TS more and more.